dev/CommonStyles/AppBarButton_rs1_themeresources.xaml

```diff
@@ -62,8 +67,10 @@
         <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
         <Setter Property="FontWeight" Value="Normal" />
         <Setter Property="Width" Value="68" />
-        <Setter Property="UseSystemFocusVisuals" Value="True" />
+        <contract6Present:Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
+        <contract6NotPresent:Setter Property="UseSystemFocusVisuals" Value="True" />
```

Comment:
Can we just provide the UseSystemFocusVisuals in Common_themeresources.xaml (with a conditional) so we don't need to use a conditional here?